### PR TITLE
Add new queue info - high2/med2/low2 

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

**Comment by ctb** [comment]
@taylorreiter @bluegenes review pls?

File: partitions.md [hunk 1]
```diff
@@ -17,11 +17,15 @@ Without specifying a partition, I've been able to allocate 30 GB but not
...
-* bmm - medium priority big mem
-* bmh - high priority big mem
+* bml - low priority big mem node (up to a TB)
+* bmm - medium priority big mem node
+* bmh - high priority big mem node
 
-as well as some default queue/partition that I don't yet understand :)
+as well as
+
+* low2 - low priority compute nodes
```
Comment (shannonekj) [inline partitions.md:26]: add "(up to 256Gb)"

Comment (shannonekj) [reply partitions.md:26]: I see you specified below, so I suggest adding or deleting the (up to a TB) on line 20

File: getting-started.md [hunk 2]
```diff
@@ -179,12 +181,17 @@ For files shared among users (references, databases, etc), use /group/ctbrowngrp
...
 
 Users in `ctbrowngrp` collectively share resources. 
-Currently, this group has priority access to 1 TB of ram and 96 CPUs.
-These resources are accessed using the big mem partition, `bm*`.
+Currently, this group has priority access to 1 TB of ram and 96 CPUs on
+one machine, and 256 GB RAM and up to 64 CPUs on another two machines.
+The big mem machine is accessed using the big mem partition, `bm*`,
+while the smaller-memory machines are accessed on `high2`/`med2`/`low2`.
+
 As of February 2020, there are 31 researches who share these resources.
```
Comment (shannonekj) [inline getting-started.md:189]: change researches to researchers

File: getting-started.md [hunk 3]
```diff
@@ -179,12 +181,17 @@ For files shared among users (references, databases, etc), use /group/ctbrowngrp
 ## Using shared resources
 
 Users in `ctbrowngrp` collectively share resources. 
-Currently, this group has priority access to 1 TB of ram and 96 CPUs.
-These resources are accessed using the big mem partition, `bm*`.
+Currently, this group has priority access to 1 TB of ram and 96 CPUs on
+one machine, and 256 GB RAM and up to 64 CPUs on another two machines.
```
Comment (shannonekj) [inline getting-started.md:185]: suggest change to "Currently, this group has priority access to 1 TB of ram and 96 CPUs on
one big mem machine..."

**Pull request merged**